modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/OpenEventCollector.java

```diff
@@ -106,8 +107,15 @@ public static Integer reportEntryEvent(MessageContext messageContext, String com
 
 			StatisticsOpenEvent openEvent = new StatisticsOpenEvent(statisticDataUnit);
             addEventAndIncrementCount(messageContext, openEvent);
+
+            if (isOpenTracingEnabled()) {
+				OpenTracingManagerHolder.getOpenTracingManager().getHandler()
+						.handleOpenEntryEvent(statisticDataUnit, messageContext);
+			}
```

Comment:
Formatting issue.

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/f667809fdde325456df56a477b83e6ef731107ca